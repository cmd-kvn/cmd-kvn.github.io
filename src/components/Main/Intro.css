article {
    padding: 8px;
}