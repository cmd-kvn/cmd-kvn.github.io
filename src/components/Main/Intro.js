import React from 'react';
import Markdown from 'markdown-to-jsx';

import './Intro.css';

const markdown = `
## Hello! I'm Kevin - your friendly neighborhood JavaScript developer.  
When I'm not protecting the community from \`eslint\` errors and 
unresolved \`Promise\` chains you will find me volunteering, at your 
local Meetup, or dancing Cuban salsa!
`.trim();

const Intro = () => {
    return (
        <article>
            <Markdown>
                {markdown}
            </Markdown>
        </article>
    );
};

export default Intro;
