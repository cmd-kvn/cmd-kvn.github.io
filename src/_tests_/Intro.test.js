import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Intro from '../components/Main/Intro';

describe('Intro', function() {
    it('runs the Intro', function() {
        const rendered = shallow(<Intro />);
        expect(toJson(rendered)).toMatchSnapshot();
    });
});
