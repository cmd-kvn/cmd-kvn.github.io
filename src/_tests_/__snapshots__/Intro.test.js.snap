exports[`Intro runs the Intro 1`] = `
<article>
  <Component>
    ## Hello! I\'m Kevin - your friendly neighborhood JavaScript developer.  
    When I\'m not protecting the community from \`eslint\` errors and 
    unresolved \`Promise\` chains you will find me volunteering, at your 
    local Meetup, or dancing Cuban salsa!
  </Component>
</article>
`;
